### ISSUE TITLE
The code running results are inconsistent with the experiment

### ISSUE THREAD

**Issue by iffuture799** [raymin0223/patch-mix_contrastive_learning#3]
I tried running icbhi_patchmix_cl.sh file, but I found that the running result is only around 60.2(Best S_p 77.14, S_e 43.25, Score 60.19). I am not sure what the problem is. I hope someone can help me. Thank you very much

**Comment by raymin0223** [comment]
Hi @iffuture799,

We have also found the results in certain seeds show relatively lower performance (but, not that much like 60.19).   
I believe that the reason is the inherent property of the ICBHI dataset, itself.
Thus, I recommend you to run on different seeds, as the hyperparameters are exactly same with `icbhi_patchmix_cl.sh`.

**Comment by iffuture799** [comment]
Thank you very much for your reply. I will try running it again

**Comment by iffuture799** [comment]
Hi @raymin0223 ，
Excuse me, I would like to ask some more detailed questions. What model of Gpu do you use to run the code, and what versions of Python, Torchvision, Torchaudio, and Cuda do you use. I tried running the code using different seeds but still couldn't get the results in the paper.

**Comment by raymin0223** [comment]
As I run the codes again with two different seeds, I got the results like these figures.
<img width="631" alt="스크린샷 2023-09-06 오후 8 56 00" src="https://github.com/raymin0223/patch-mix_contrastive_learning/assets/50742281/5c839ee0-342e-4abe-bddf-39a6a7a3505e">
<img width="610" alt="스크린샷 2023-09-06 오후 11 24 19" src="https://github.com/raymin0223/patch-mix_contrastive_learning/assets/50742281/79968c53-972a-4149-8d30-5588d958c19f">

GPU is A5000, `torch` version is '2.0.1+cu117', `torchvision` is '0.15.2+cu117', and `torchaudio` is '2.0.2+cu117'.

Our previous experimental results was like (81.70, 42.82, **62.26**), (77.20, 45,79, **61.50**), (83.72, 41.21, **62.47**), (78.85, 45.96, **62.41**), (86.83, 39.59, **63.21**).
I think that new results are within the bound of reported results, but it highly depends on the experiment seeds and I might have mistakes during refactoring.

**Comment by iffuture799** [comment]
Thank you very much for your patience in answering my question. Your reply will be of great help to me

**Comment by raymin0223** [comment]
No problem and thanks so much for your interests to our work :)
If you have further questions, feel free to ask us or reopen the issues!

**Comment by iffuture799** [comment]
Hi @raymin0223 ，

I ran the code and the result is still around 60. Can you share the dataset being used and the pre trained model? I think this is the only difference between me and you.

**Comment by raymin0223** [comment]
I downloaded ICBHI dataset from scratch by using the following command:
`$ wget https://bhichallenge.med.auth.gr/sites/default/files/ICBHI_final_database/ICBHI_final_database.zip`

When running the code, you should get the same data distribution like this,
![스크린샷 2023-09-07 오후 9 58 31](https://github.com/raymin0223/patch-mix_contrastive_learning/assets/50742281/2a9a715a-2a45-4816-ba99-ad0096ed37b6)

At this time, I got (86.19, 38.15, 62.17) score, and you can download ckpt in [here](https://drive.google.com/file/d/17_RT3mG0xLSicE4Tm9qaB1rfIxcTYcNJ/view?usp=share_link)
![스크린샷 2023-09-08 오전 1 06 19](https://github.com/raymin0223/patch-mix_contrastive_learning/assets/50742281/78151a88-6916-43db-b5d4-ce5a01f24632)

**Comment by iffuture799** [comment]
Thank you！:)

**Comment by chumingqian** [comment]
Hi, @iffuture799 ，@raymin0223
 During  the  processing ,  i found  the  batch size  can   affect  the   accuracy;
